### ISSUE TITLE
make launch C++17 error windows 

### ISSUE THREAD

**Issue by savinovava** [carla-simulator/carla#2499]
![image](https://user-images.githubusercontent.com/53625116/75033176-62b70300-54bb-11ea-8330-f6b5d851b336.png)
```
 LocalizationStage.cpp
  class template optional is only available with C++17 or later.
and after a lot of errors,
```
i'm using VS 2017

![image](https://user-images.githubusercontent.com/53625116/75033387-d78a3d00-54bb-11ea-8e64-6f87062d233a.png)

I have downloaded C++/WinRT but it doesn't help

**Comment by jackbart94** [comment]
Hi,

Sorry for the issue, one of the variables is declared as `uint` instead of `uint64_t`. It's on the line of the error, changing that should solve your problem.

**Comment by JimWhiteheadMR** [comment]
Just a quick note to say that I also ran into this same issue (uint instead of specific width integer) with LocalizationStage.cpp, and in my case changing all instances of uint to uint32_t in LocalizationStage.cpp fixed the problem.

I also ran into a similar issue with ProfilesParser.cpp. Here I changed all variable declarations of uint to uint32_t, and this fixed the issue. Note that in ProfilesParser.cpp, I did not change lines of the form:

node_road.attribute("id").as_uint();

These were left as-is, since there is no as_uint32_t() equivalent.

I am compiling on Windows 10, using Visual Studio 2017.
